src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -454,11 +454,11 @@ private function resolveServices($value)
             $value = array_map(array($this, 'resolveServices'), $value);
         } elseif (is_string($value) &&  0 === strpos($value, '@=')) {
             return new Expression(substr($value, 2));
-        } elseif (is_string($value) &&  0 === strpos($value, '@')) {
-            if (0 === strpos($value, '@@')) {
+        } elseif (is_string($value) &&  $value[0] === '@') {
```

Comment:
Yep, that would generate a Notice.
Can the case happen ? The problem of strpos() is that it is a function call (heavy) and it has to scan the whole string to find a char.

We may add if ($str[0] && $str[0] == '@') {  in such a case